src/test/java/gov/cms/madiejavamodels/MeasurePopulationTest.java

```diff
@@ -0,0 +1,33 @@
+package gov.cms.madiejavamodels;
+
+import gov.cms.madiejavamodels.measure.MeasurePopulation;
+import org.junit.Test;
+
+
+public class MeasurePopulationTest {
+  @Test
+  public void testPopulationToCode() {
+    assert MeasurePopulation.INITIAL_POPULATION.toCode() == "initial-population";
```

Comment:
`assert` is generally not recommend for testing since it requires a JVM flag to run and is otherwise skipped.

Reply:
updated